Tests/ARTRealtimeMessageTest.m

```diff
@@ -58,7 +58,8 @@ - (void)multipleSendName:(NSString *)name count:(int)count delay:(int)delay {
             }];
         }
     }];
-    [self waitForExpectationsWithTimeout:[ARTTestUtil timeout]+delay handler:nil];
+    [self waitForExpectationsWithTimeout:50.0 handler:nil];
```

Comment:
It seems that `SocketRocket` is slower than `SwiftWebSocket`.
Total time increased from 18s seconds to 47s.